openedx/core/djangoapps/user_api/tests/test_helpers.py

```diff
@@ -214,3 +216,63 @@ def stub_view(request):  # pylint: disable=missing-docstring
             self.captured_request = request
             return response
         return shim_student_view(stub_view, check_logged_in=check_logged_in)
+
+
+class DummyRegistrationExtensionModel(object):
+    """
+    Dummy registration object
+    """
+    user = None
+
+    def save(self):
+        """
+        Dummy save method for dummy model.
+        """
+        return None
+
+
+class TestCaseForm(forms.Form):
+    """
+    Test registration extension form.
+    """
+    DUMMY_STORAGE = {}
+
+    MOVIE_MIN_LEN = 3
+    MOVIE_MAX_LEN = 100
+
+    FAVORITE_EDITOR = (
+        ('vim', 'Vim'),
+        ('emacs', 'Emacs'),
+        ('np', 'Notepad'),
+        ('cat', 'cat > filename')
+    )
+
+    favorite_movie = forms.CharField(
+        label="Fav Flick", min_length=MOVIE_MIN_LEN, max_length=MOVIE_MAX_LEN, error_messages={
+            "required": u"Please tell us your favorite movie.",
+            "invalid": u"We're pretty sure you made that movie up."
+        }
+    )
+    favorite_editor = forms.ChoiceField(label="Favorite Editor", choices=FAVORITE_EDITOR, required=False, initial='cat')
+
+    # pylint: disable=unused-argument
```

Comment:
Nit: putting the pragma on the same line only disables it for that line. This disables it for the rest of the class.